CONSOLE-4272: report csp violations to telemtry (#14486)
openshift/console: README.md

docker run -p 8081:8081 downloadsserver:latest
```

## ContentSecurityViolation Detection

> The console app includes code which automatically reports CSP violoations to telemetry. This

Reviewer: ```suggestion
CSP violations are automatically reported to telemetry through the console application. This
```
Can we update to say something like this? or
```suggestion
The console application automatically reports CSP violations to telemetry. This
```
I don't think you need to say "includes code" it is kind of implied.

> detection and reporting logic tries to parse a dynamic plugin name from the securitypolicyviolation

Reviewer: ```suggestion
detection and reporting logic attempts to parse a dynamic plugin name from the securitypolicyviolation
```

> event to include in the data reported to telemetry. If we are unable to determine a plugin name in

Reviewer: ```suggestion
event to include in the data reported to telemetry. If a plugin name is not determined in
```

> this way, then 'none' will be used. We also throttle violation reporting In order to prevent

Reviewer: ```suggestion
this way, then 'none' will be used. Additionally, violation reporting is throttled to prevent
```

> spamming our  telemetry service with repetitive data. This means identical violations will not be

Reviewer: ```suggestion
spamming the telemetry service with repetitive data. Identical violations will not be
```

> reported more than once per day.

Reviewer: ```suggestion
reported more than once a day.
```
